### PR TITLE
[CRT] Remove mingwm10.dll usage in tls support code

### PR DESCRIPTION
**Pull request by learn-more**
JIRA issue: [CORE-15491](https://jira.reactos.org/browse/CORE-15491)

### CODE REVIEW

File: sdk/lib/crt/startup/tlsmthread.c [hunk 1]
```diff
@@ -20,11 +20,6 @@ extern int ___w64_mingwthr_add_key_dtor (DWORD key, void (*dtor)(void *));
```
Comment (HBelusca) [inline sdk/lib/crt/startup/tlsmthread.c]: But all the rest in this file is still needed? o_O

Comment (learn-more) [reply sdk/lib/crt/startup/tlsmthread.c]: fix gcc.

Comment (HBelusca) [reply sdk/lib/crt/startup/tlsmthread.c]: OK no problem, I just want also to be sure that this file is not used at all when compiling with MSVC.

**Comment by yagoulas** [comment]
why on earth was that thing there?

**Comment by learn-more** [comment]
Probably because this is from the mingw runtime...

**Pull request merged**